sdk/appconfiguration/app-configuration/test/utils/testHelpers.ts

```diff
@@ -70,8 +69,8 @@ export function getTokenAuthenticationCredential(): CredsAndEndpoint | undefined
   };
 }
 
-export function createAppConfigurationClientForTests(
-  options?: InternalAppConfigurationClientOptions
+export function createAppConfigurationClientForTests<Options extends AppConfigurationClientOptions>(
```

Comment:
I don't know if it really matters for our tests, but you could specify a default value so that you don't _have_ to specify a type here when calling this function:
```suggestion
export function createAppConfigurationClientForTests<Options extends AppConfigurationClientOptions = AppConfigurationClientOptions>(
```